Examples/Blink/Core/Src/main.cpp

```diff
@@ -19,8 +19,10 @@
 /* USER CODE END Header */

```

Comment:
Probably wouldn't hurt to add some testing in to the main here, look to Akash's SPI interface PR to see what he did, your testing will be very similar.

Reply:
100% I'll add this to my TODO list